packages/shared/src/regex.ts

```diff
@@ -2,6 +2,6 @@ export const emailRegEx = /^\S+@\S+\.\S+$/;
 export const phoneRegEx = /^\d+$/;
 export const usernameRegEx = /^[A-Z_a-z]\w*$/;
 export const passwordRegEx = /^.{6,}$/;
-export const redirectUriRegEx = /^https?:\/\//;
+export const redirectUriRegEx = /^https?:\/\/.*|^[a-z][\d_a-z]*(\.[\d_a-z]+)+[\d_a-z]:\/\/[a-z]*$/;
```

Comment:
The regex should allow this URI format: `io.example://abc/callback`
But currently it fails the validation

Reply:
<img width="265" alt="image" src="https://user-images.githubusercontent.com/12833674/189409178-2d62a2ae-462f-4ea9-8d6e-6be04126eda4.png">
I tried but this is still not allowed for native apps, but it should